### ISSUE TITLE
project fails to compile with externs in the resources folder

### ISSUE THREAD

**Issue by vegegoku** [Vertispan/j2clmavenplugin#71]
If we have a project that include the elemental2-svg dependency and we include the svg extern file in the project resources directory the plugin fails to load the extern to compile the project:

```
/mnt/CODE/dominokit-testing/cli-test/testj2clbasic/testj2clbasic-client/target/gwt3BuildCache/0.16-SNAPSHOT/testj2clbasic-client-d9552c9457161f08de5041e50b8ba831a919691f/closure-inputs/elemental2/svg/SVGAElement$$Overlay.impl.java.js:13: ERROR - [JSC_UNDEFINED_VARIABLE] variable SVGAElement is undeclared
  13|   return instance instanceof SVGAElement;
                                   ^^^^^^^^^^^

/mnt/CODE/dominokit-testing/cli-test/testj2clbasic/testj2clbasic-client/target/gwt3BuildCache/0.16-SNAPSHOT/testj2clbasic-client-d9552c9457161f08de5041e50b8ba831a919691f/closure-inputs/elemental2/svg/SVGAltGlyphDefElement$$Overlay.impl.java.js:13: ERROR - [JSC_UNDEFINED_VARIABLE] variable SVGAltGlyphDefElement is undeclared
  13|   return instance instanceof SVGAltGlyphDefElement;
                                   ^^^^^^^^^^^^^^^^^^^^^

/mnt/CODE/dominokit-testing/cli-test/testj2clbasic/testj2clbasic-client/target/gwt3BuildCache/0.16-SNAPSHOT/testj2clbasic-client-d9552c9457161f08de5041e50b8ba831a919691f/closure-inputs/elemental2/svg/SVGAltGlyphElement$$Overlay.impl.java.js:13: ERROR - [JSC_UNDEFINED_VARIABLE] variable SVGAltGlyphElement is undeclared
  13|   return instance instanceof SVGAltGlyphElement;
                                   ^^^^^^^^^^^^^^^^^^

/mnt/CODE/dominokit-testing/cli-test/testj2clbasic/testj2clbasic-client/target/gwt3BuildCache/0.16-SNAPSHOT/testj2clbasic-client-d9552c9457161f08de5041e50b8ba831a919691f/closure-inputs/elemental2/svg/SVGAltGlyphItemElement$$Overlay.impl.java.js:13: ERROR - [JSC_UNDEFINED_VARIABLE] variable SVGAltGlyphItemElement is undeclared
  13|   return instance instanceof SVGAltGlyphItemElement;
                                   ^^^^^^^^^^^^^^^^^^^^^^

/mnt/CODE/dominokit-testing/cli-test/testj2clbasic/testj2clbasic-client/target/gwt3BuildCache/0.16-SNAPSHOT/testj2clbasic-client-d9552c9457161f08de5041e50b8ba831a919691f/closure-inputs/elemental2/svg/SVGAngle$$Overlay.impl.java.js:30: 
Originally at:
/mnt/CODE/dominokit-testing/cli-test/testj2clbasic/testj2clbasic-client/target/gwt3BuildCache/0.16-SNAPSHOT/testj2clbasic-client-d9552c9457161f08de5041e50b8ba831a919691f/closure-inputs/elemental2/svg/SVGAngle.java:24: ERROR - [JSC_UNDEFINED_VARIABLE] variable SVGAngle is undeclared
  24|   @JsOverlay public static final double SVG_ANGLETYPE_DEG = SVGAngle__Constants.SVG_ANGLETYPE_DEG;
        ^^^^^^^^

/mnt/CODE/dominokit-testing/cli-test/testj2clbasic/testj2clbasic-client/target/gwt3BuildCache/0.16-SNAPSHOT/testj2clbasic-client-d9552c9457161f08de5041e50b8ba831a919691f/closure-inputs/elemental2/svg/SVGAnimateColorElement$$Overlay.impl.java.js:13: ERROR - [JSC_UNDEFINED_VARIABLE] variable SVGAnimateColorElement is undeclared
  13|   return instance instanceof SVGAnimateColorElement;
                                   ^^^^^^^^^^^^^^^^^^^^^^

/mnt/CODE/dominokit-testing/cli-test/testj2clbasic/testj2clbasic-client/target/gwt3BuildCache/0.16-SNAPSHOT/testj2clbasic-client-d9552c9457161f08de5041e50b8ba831a919691f/closure-inputs/elemental2/svg/SVGAnimateElement$$Overlay.impl.java.js:13: ERROR - [JSC_UNDEFINED_VARIABLE] variable SVGAnimateElement is undeclared
  13|   return instance instanceof SVGAnimateElement;
                                   ^^^^^^^^^^^^^^^^^

/mnt/CODE/dominokit-testing/cli-test/testj2clbasic/testj2clbasic-client/target/gwt3BuildCache/0.16-SNAPSHOT/testj2clbasic-client-d9552c9457161f08de5041e50b8ba831a919691f/closure-inputs/elemental2/svg/SVGAnimateMotionElement$$Overlay.impl.java.js:13: ERROR - [JSC_UNDEFINED_VARIABLE] variable SVGAnimateMotionElement is undeclared
  13|   return instance instanceof SVGAnimateMotionElement;
                                   ^^^^^^^^^^^^^^^^^^^^^^^
```

the extern for svg can be found here https://github.com/google/closure-compiler/blob/master/contrib/externs/svg.js

[testj2clbasic.zip](https://github.com/Vertispan/j2clmavenplugin/files/6228698/testj2clbasic.zip)

attached is a project to reproduce the issue

**Comment by niloc132** [comment]
This is the same bug as #83, fixed by 277714706fc.

**Comment by niloc132** [comment]
This is the same bug as #83, fixed by 277714706fc.
